blocks/article-hero-video/article-hero-video.js

```diff
@@ -0,0 +1,71 @@
+import { readBlockConfig } from '../../scripts/lib-franklin.js';
+
+export default function decorate(block) {
+  const data = readBlockConfig(block);
+  // Get the image element from the block before we clear it
+  const picture = block.querySelector('picture');
+  block.innerText = '';
+
+  if (data?.video) {
+    block.append(buildIframe(data?.video));
+  }
+
+  // create image container div
+  const imageContainer = document.createElement('div');
+  imageContainer.classList.add('article-hero-video-image-container');
+  picture.classList.add('article-hero-video-image');
+  imageContainer.append(picture);
+  // append image container to block
+  block.append(imageContainer);
+
+  const titleContainer = document.createElement('div');
+  titleContainer.classList.add('article-hero-video-title-container');
+
+  const section = document.createElement('h4');
+  section.classList.add('article-hero-video-section');
+  section.innerText = data?.section;
```

Comment:
Let's infer the section from the document metadata, you can use `getMetadata('section')?.toLowerCase()` (imported from scripts.js)
And you can remove it from the block content.

Reply:
Done